test/utils.js

```diff
@@ -110,15 +103,20 @@ function getWsUrlWithControlAndMessageLayerVersions(port, ssl = false, controlLa
     return `${ssl ? 'wss' : 'ws'}://127.0.0.1:${port}/api/v1/ws?controlLayerVersion=${controlLayerVersion}&messageLayerVersion=${messageLayerVersion}`
 }
 
-function createClient(wsPort, clientOptions = DEFAULT_CLIENT_OPTIONS) {
+const createMockUser = () => ethers.Wallet.createRandom()
```

Comment:
About terminology, `user` vs. `wallet` vs. `privateKey`?

Reply:
In tests we handle this object as a user account: e.g. we grant permissions to a user (`broker.test.js`), and create streams with valid usename prefix (e.g. `DeleteExpiredCmd.test.js`). Backend creates a new user account for each of this objects.